collection.php

```diff
@@ -0,0 +1,34 @@
+<?php
+//--QUERY--
+$db = new PDO('mysql:host=db; dbname=collection', 'root', 'password');
+$query = $db->prepare("SELECT `setName`, `setNumber`, `pieceCount`, `releaseYear` FROM `sets`;");
+$query->execute();
+$sets = $query->fetchAll();
```

Comment:
I would also suggest popping this chunk into its own function for getting the collection.